### ISSUE TITLE
Replace loop for software installation

### ISSUE THREAD

**Issue by pvanheus** [SANBI-SA/ansible-role-common#10]
https://github.com/SANBI-SA/ansible-role-common/blob/master/tasks/00_software.yml#L11 - causes a deprecation warning in Ansible, needs replacement.

**Comment by pvanheus** [comment]
The line number now appears to be wrong. Perhaps this issue relates to this warning:

```
TASK [sanbi-sa.common : Install base software set] ************************************************************************************************************************************************
[DEPRECATION WARNING]: Invoking "apt" only once while using a loop via squash_actions is deprecated. Instead of using a loop to supply multiple items and specifying `name: "{{ item }}"`, please 
use `name: ['{{ ceph_packages }}', 'unzip', 'ntp']` and remove the loop. This feature will be removed in version 2.11. Deprecation warnings can be disabled by setting deprecation_warnings=False 
in ansible.cfg.
```

**Comment by Banshee1221** [comment]
this is fixed by #19
